Source/Core/Core/IOS/ES/TitleManagement.cpp

```diff
@@ -438,25 +458,10 @@ ReturnCode ES::ImportTitleDone(Context& context)
     return ES_EINVAL;
   }
 
-  // Make sure all listed, non-optional contents have been imported.
+  // For system titles, make sure all listed, non-optional contents have been imported.
   const u64 title_id = context.title_import_export.tmd.GetTitleId();
-  const std::vector<IOS::ES::Content> contents = context.title_import_export.tmd.GetContents();
-  const IOS::ES::SharedContentMap shared_content_map{m_ios.GetFS()};
-  const bool has_all_required_contents =
-      std::all_of(contents.cbegin(), contents.cend(), [&](const IOS::ES::Content& content) {
-        if (content.IsOptional())
-          return true;
-
-        if (content.IsShared())
-          return shared_content_map.GetFilenameFromSHA1(content.sha1).has_value();
-
-        // Note: the import hasn't been finalised yet, so the whole title directory
-        // is still in /import, not /title.
-        const std::string path = Common::GetImportTitlePath(title_id) +
-                                 StringFromFormat("/content/%08x.app", content.id);
-        return m_ios.GetFS()->GetMetadata(PID_KERNEL, PID_KERNEL, path).Succeeded();
-      });
-  if (!has_all_required_contents)
+  if (title_id - 0x100000001LL <= 0x100 &&
```

Comment:
Don't we have a constant somewhere for 1-1?

Reply:
We have one for 1-2 (the System Menu), but none for 1-1 (that title doesn't seem to exist?)

Reply:
It's what ES checks, not sure why Nintendo bothered checking the TID falls exactly in the range [1-1, 1-101] and didn't just check the higher 32 bits (title type).

Reply:
Good enough for me then.